docker/chromium/Dockerfile

```diff
@@ -19,7 +21,9 @@ RUN apt-get update && apt-get upgrade -y && apt-get install -y --no-install-reco
 && apt-get purge -y --auto-remove \
   python3-dev \
   zlib1g-dev \
-&& rm -rf /var/lib/apt/lists/*
+&& rm -rf /var/lib/apt/lists/* \
+&& chmod +x ./start_headless.sh \
```

Comment:
this script doesnt seem to exist

Reply:
[Removed](https://github.com/demisto/dockerfiles/pull/21881/commits/9f46c09f21312272f38c4bde33980c5e6bcf4613).